src/components/Header/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import "./Header.scss";
+import { Link } from "react-router-dom";
+
+type HeaderProps = {
+  title: string;
+  subtitle?: string;
+  profileImage?: string;
+};
+
+const Header = ({ title, subtitle, profileImage }: HeaderProps) => {
+  return (
+    <header className="header">
+      <div className="header__image-div">
+        <Link to="/profile">
+          <img
+            className="header__image-div header__image-div--image"
+            src={profileImage}
```

Comment:
I think that we need to handle if the profileImage is not passed as a prop, perhaps we have a default image to use if it is undefined given?